frontend/src/components/StockDialog.js

```diff
@@ -71,12 +71,14 @@ const StockDialog = (props) => {
         <span style={{ fontSize: "0.8rem", color: "rgb(128, 128, 128)" }}>
           Date: {day} {month}
         </span>
-        <Button onClick={handleClose} style={{marginLeft:"300px"}} autoFocus>
```

Comment:
Fixes this:

![image](https://github.com/dkaneria14/Web-scraper/assets/23105842/9070c085-2a29-4d01-8798-7bf749939250)